libreflexxestype2/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(libreflexxestype2)
 
-# Compile as C++14, supported in ROS Melodic and newer
-set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED YES)
```

Comment:
If we still want to set the language standard, which we should, lets use `target_compile_features` to do so. 

```cmake
target_compile_features(reflexxes PUBLIC cxx_std_14)
```

Doing it this way ensures that language requirements are transitively passed on to downstream targets.

Reply:
you should not set language standards, the compiler has a sane default.

Reply:
You should absolutely set the language requirements. If the compiler default matches what you require then the default is used. If a compiler supports C++14 but defaults to C++11 then we want to make sure the default isn't used. It's widely regarded as CMake best practices to specify language standard requirements.